### PR TITLE
Title fix + non-scrollable fix

### PR DESCRIPTION
**Pull request by colinfruit**

### CODE REVIEW

**Comment by colinfruit** [comment]
I guess this finishes #179

File: src/components/Content/RightPanel/TaskDetails/TaskTitle/TaskTitle.js [hunk 1]
```diff
@@ -3,6 +3,12 @@ import React, { Component } from 'react';
 import './TaskTitle.css';
 
 class TaskTitle extends Component {
+  static autoResize() {
+    var el = document.getElementById('discuss-task-title-update');
+    el.style.height = "";
+    el.style.height = (el.scrollHeight + 2) + "px";
+  }
+
```
Comment (elimisteve) [inline src/components/Content/RightPanel/TaskDetails/TaskTitle/TaskTitle.js:11]: @colinfruit To clarify my comments earlier, seems like we should be able to set the height using CSS instead here :+1:

File: src/components/NavBar/LogIn/LogIn.js [hunk 2]
```diff
@@ -56,6 +56,8 @@ class LogIn extends Component {
     this.usernameRemembered = '';
     this.username = '';
     this.password = '';
+    // remove modal class from body.
+    document.body.className = document.body.className.replace('modal-open','');
```
Comment (elimisteve) [inline src/components/NavBar/LogIn/LogIn.js:60]: @colinfruit Does this cause the modal to close?

**Comment by elimisteve** [comment]
@colinfruit Merged! This works -- great job! -- so it's merged for now and if we run into issues with React clobbering the DOM mutations we can look back into this.

But at the moment, we have much bigger fish to fry :fist_raised:

**Pull request merged**